assets/dev/scss/editor/_global.scss

```diff
@@ -166,3 +165,31 @@
 i.eicon-tilted {
 	transform: rotate(90deg);
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
+	margin-top: 15px;
+}
+
+.elementor-control-media-size .elementor-control-field {
+	padding: 0px;
+}
+
+.elementor-control-media-size .elementor-control-responsive-switchers {
+	display: none;
+}
+
+.elementor-control-media-display-size.select {
+	padding-bottom: 0px;
+	width: 72%;
+	float: right;
+}
+
+.media-size-input-wrapper {
+	position: unset;
```

Comment:
why?

Reply:
UI issue from the wrapper CSS

Reply:
yeah, but explain the issue in the code. Plus, which issue can a position cause?

Reply:
`position` removed. It was unnecessary after all.